docs/api/core/InterleavedBuffer.html

```diff
@@ -11,21 +11,34 @@
 		<h1>[name]</h1>
 
 		<div class="desc">
+			"Interleaved" means that multiple types of data (position, color etc.) are packed into a single array buffer.
+			Organizing geometry data in this way improves memory locality. By keeping vertex data close together, it is more likely
+			that also caches (e.g. pre-transform vertex cache) store logically related data together. This leads to more cache hits and a better performance.
```

Comment:
Nice PR, but maybe you can rephrase or remove these two sentences...

Reply:
Can you make a suggestion for an alternative? I thought it might be a good Idea to highlight the primary motivation for using interleaved buffers.

Reply:
I would write it this way:

>"Interleaved" means that multiple attributes, possibly of different types, (e.g., position, normal, uv, color) are packed into a single array buffer.

I would leave it at that and avoid a tutorial. Also, avoid making performance claims.

>I thought it might be a good Idea to highlight the primary motivation for using interleaved buffers.

What is your reference for the last two sentences? Is that the "primary motivation"?

Reply:
Okay, i updated the docs with your suggestion. It's okay for me to exclude the other stuff.

> What is your reference for the last two sentences? Is that the "primary motivation"? (For a moment i forgot to focus just on docs and not tutorials :innocent:).

It is mentioned in this [literature](http://www.wrox.com/WileyCDA/WroxTitle/Professional-WebGL-Programming-Developing-3D-Graphics-for-the-Web.productCd-1119968860.html) , Chapter 3, _Interleaving your vertex data for improved performance_.

This is the section i refer to: _From a performance point of view, the preferred way to organize your vertex data is to interleave the data as an array of structures. This is because you get better memory locality four your vertex data. It is likely that when a vertex shader needs the position of a certain vertex, it will also ned the normal and texture coordinates for the same vertex at almost the same time. By keeping these coordinates close together in the memory, it is more likely that when one is read into the pre-transform vertex cache, the others will also be read in the same block and will be available in the pre-transform vertex cache when they are needed by the vertex shader._

Well, "primary motivation" is actually my personal wording 😉 . But i think it's okay to say that performance is the main goal for when people interleaving vertex data.